ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.hbs

```diff
@@ -67,12 +67,15 @@
         data-test-button="preview"
       />
 
-      <Hds::Button
-        @text="{{if @message.isNew 'Create' 'Edit'}} message"
-        disabled={{(or this.invalidFormMessage this.save.isRunning)}}
-        data-test-button="create-message"
-        type="submit"
-      />
+      {{#if this.hasSomeActiveModals}}
+        <Hds::Button
```

Comment:
If we move the active modal check to within the save method so that it can continue seamlessly on confirm, we won't need this if statement